### PR TITLE
Edit readme api

### PR DESCRIPTION
**Pull request by kdough03**
Made some formatting changes, including formatting code blocks and a table. I'll submit more PRs in the future, but didn't want to make too many changes at once.

### CODE REVIEW

File: README_API.md [hunk 1]
```diff
@@ -644,216 +677,253 @@ curl -X PUT -d "lat=54.33&lon=-1.467&x=3666.335&y=2000.12&format=json" -u user@e
...
-=== Crop - Save, Mask and Warp Map  ===
-Requires authentication
+**Definition**
 
+```
 POST http://mapwarper.net/maps/{map_id}/save_mask_and_warp
+```
 
-rolls the calls into one. Saves mask, applies mask to map, and rectifies map using the mask
+???????????????? PARAMETERS ???????????????
```
Comment (timwaters) [inline README_API.md]: is this ????? a todo

**Comment by timwaters** [comment]
many thanks, I like the formatting of the tables. I think there's just that one line which I've commented on which may have been missed.

**Comment by timwaters** [comment]
Also, if possible, could these commits be squished into one? It's not essential & won't break anything but instead of having 19 commit entries it would be the 1 commit for this change making the commit history easier to read and for people to easier understand the changes. 

Generally speaking it doesn't necessarily need to be down to one commit though (for example, where a PR fixes three bugs at once, it could be good to have the commits squashed to three commits).

File: README_API.md (hunk 1 above)
Comment (kdough03) [inline README_API.md]: Yes.

**Comment by kdough03** [comment]
Sure, I could make all the minor changes at once, then paste it into a new branch and send a unified PR with one commit (unless there's a better way to do this). Feel free to close out the existing PRs.

**Comment by timwaters** [comment]
Yeah that would work - there is also a "git" way of doing it using git rebase also But I don't know if it would require a new PR. I have to confess I've not used github's pull requests that much!

These could help:
http://blog.steveklabnik.com/posts/2012-11-08-how-to-squash-commits-in-a-github-pull-request is more concise than 
https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request

**Comment by Abbe98** [comment]
@timwaters https://github.com/blog/2141-squash-your-commits

**Comment by timwaters** [comment]
Ah ha! That's interesting!

File: README_API.md [hunk 2]
```diff
@@ -822,9 +834,11 @@ format  jsonoutput     a GML string containing for example:
 ```
 
 **Response**
-Returns a text string with a message indicating success or failure.
+
+A successful call will return the following message. 
+
 ```
-{"stat":"ok", "message":"Map clipping mask saved (gml)"}
+{"message":"Map clipping mask saved (gml)"}
```
Comment (timwaters) [inline README_API.md]: seems like this is missing the {"stat":"ok" part ?

**Comment by timwaters** [comment]
okay so for squashing it looks as if me, via github can do this. 
Looking over the commits, there appears to be a couple of todos etc 

Let me know when all is finished and I can merge in this PR

**Comment by kdough03** [comment]
You can go ahead and delete/close the PR. I'll submit a new one from a 
new branch, and it will be clean - I'll submit the "to dos" separately.


On 4/16/16 12:45 PM, Tim Waters wrote:
>
> okay so for squashing it looks as if me, via github can do this.
> Looking over the commits, there appears to be a couple of todos etc
>
> Let me know when all is finished and I can merge in this PR
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/timwaters/mapwarper/pull/89#issuecomment-210853855>
>

-- 

Kate Dougherty
Freelance Technical Writer, Science Writer, and Researcher
(267) 357-0149 | Skype: kate_dougherty
http://katedoughertywriter.com/
http://www.linkedin.com/in/katedoughertywriter

**Pull request closed without merging**